pages/about/contribute.md

```diff
@@ -26,43 +26,41 @@ Please follow the prerequisites to begin the project.
         - Debian/Ubuntu: `sudo apt update && sudo apt install git`
     - Verify installation: Open a terminal window and type `git --version`.
 
-- In your terminal, navigate to the "Colib" folder: `cd ~/Documents/colib`
-
-- Clone the project : `git clone https://github.com/CRESYM/colib0.github.io.git`
-
-- Verify you are on the main branch: `git status` should return `on branch main`
+- Refer to the [README file](https://github.com/CRESYM/colib0.github.io#) to follow the installation guide and steps for setting up the website locally.
```

Comment:
i would say that this is not mandatory to emulate locally. in any case, this is more detailed afterwards so i would move it to the dedicated section.

About the local emulation, i would propose those two options:

- if you want to test your changes on an emulated local version of the website, please follow the installation guide and steps from the README file.
- if you want to emulate markdown behavior and correct your markdown page, you can download a markdown extension in your IDE (such as Markdown All in One for VSCode, or Markdown plugin for intellij)

Reply:
Incorporated review comments. Removed the description of emulation, but provided the link to the README to complete the installation guide process.